lib/bots/hackling.rb

```diff
@@ -0,0 +1,21 @@
+require "#{
```

Comment:
That description is crap.

Anyway, it doesn't really. I was just doing something else, with it, got to this point, and never bothered refactoring.